logdevice/common/configuration/nodes/ShardStateTracker.cpp

```diff
@@ -19,16 +19,20 @@ namespace {
 ShardState::Update
 transitionToNonIntermediaryState(const ShardState& shard_state) {
   ShardState::Update update{};
-  update.transition = StorageStateTransition::OVERRIDE_STATE;
-  update.conditions = Condition::FORCE;
-
-  ShardState::Update::StateOverride state_override{};
-  state_override.storage_state =
-      toNonIntermediaryState(shard_state.storage_state);
```

Comment:
I don't think this `toNonIntermediaryState` state function is used in anything useful anymore, right? If so, let's remove it.

Reply:
done